xml/System.Windows.Forms/DomainUpDown+DomainUpDownAccessibleObject.xml

```diff
@@ -120,9 +120,22 @@
         <ReturnType>System.String</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets the accessible object name.</summary>
```

Comment:
@Tanya-Solyanik I couldn't quite follow the change you requested for this summary at bug [#460793](https://devdiv.visualstudio.com/DevDiv/_workitems?id=460793&triage=true&fullScreen=false&_a=edit)

Reply:
@mairaw, I think that the description that @Tanya-Solyanik wants is, "Gets the name of the control that the accessible object describes."

Reply:
"Gets the name of the control that the accessible object describes." sounds good

Reply:
Ok, thank you both!